lib/pages/new_group/new_group.dart

```diff
@@ -259,17 +273,22 @@ class NewGroupController extends State<NewGroup>
     numberSelectedImagesNotifier.addListener(() {
       if (numberSelectedImagesNotifier.value == 1) {
         Navigator.pop(context);
-        uploadAvatarNewGroupChat(
-          matrixClient: Matrix.of(context).client,
-          entity: imagePickerController.selectedAssets.first.asset
-        );
+        avatarNotifier.value = imagePickerController.selectedAssets.first.asset;
+        removeAllImageSelected();
       }
     });
   }
 
   void showImagesPickerAction({
     required BuildContext context,
   }) async {
+    final isLoading = createRoomStateNotifier.value?.fold(
+      (left) => false, 
+      (right) => right is UploadContentLoading || right is CreateNewGroupChatLoading
```

Comment:
`left` & `right` it doesn't make sense. Let's give it another name with a clearer meaning. Ex: `failure` or `success`

Reply:
use isRight() for now